Source/KerbalCombatSystems/ShipController.cs

```diff
@@ -300,7 +304,7 @@ private IEnumerator UpdateBehaviour()
             fc.RCSVector = Vector3.zero;
 
             // Movement.
-            if (hasPropulsion && !hasWeapons && CheckWithdraw())
+            if (hasPropulsion && !hasWeapons && CheckWithdraw() && allowWithdrawal)
```

Comment:
```suggestion
            if (allowWithdrawal && hasPropulsion && !hasWeapons && CheckWithdraw())
```

This would save having to do the vector maths in CheckWithdraw if allowWithdrawal is false.